src/main/menu.ts

```diff
@@ -224,6 +224,18 @@ export default class MenuBuilder {
         },
       ],
     };
+    const subMenuHistory: MenuItemConstructorOptions = {
```

Comment:
Make sure it works on win and linux as well, this will only add the menu item on mac

Reply:
Anyone want to be the guinea pigs for this? @hhen0005 @Taz17

Reply:
Haha Mutaz Linux moment 💀

 I have a Linux laptop I’ll do a quick test now

Reply:
![image](https://user-images.githubusercontent.com/6735055/169268390-92871411-c8b8-4f8a-a28d-6d19d433ac3d.png)

This is what it looks like - just no history menu at all. See the file and edit menus for the pattern on how to make it appear in both OS's.

Also look into disabling the option when the user is not in a project - as doesn't make much sense to go home when not already home. The pattern for that is also described in menu.ts I believe

Reply:
Just updated for it to (hopefully) work on windows and linux too. Will look into the disabling soon

Reply:
Just added the disabling of the menu option when in home screen